kolibri/plugins/learn/assets/src/views/content-card-carousel/index.vue

```diff
@@ -10,46 +10,56 @@
     </div>
 
     <div :style="widthOfCarousel" class="content-carousel-controls">
-      <ui-icon-button
-      v-if="!isFirstSet"
-      icon="arrow_back"
-      size="large"
-      class="previous"
-      @click="previousSet" />
-      <ui-icon-button
-      v-if="!isLastSet"
-      icon="arrow_forward"
-      size="large"
-      class="next" @click="nextSet" />
+      <div class="previous" @click="previousSet">
```

Comment:
this works well to achieve a larger hit area

The main disadvantage to moving the click handler to a wrapper div is that the hover states no longer reflect the clickable area. Optimally, when hovering over the active area, the button hover state would be enabled.

For this pass it's fine, but something to keep in mind.